src/chainlit/frontend/src/components/organisms/chat/index.tsx

```diff
@@ -143,6 +158,16 @@ const Chat = () => {
             />
           </ErrorBoundary>
         )}
+        {!messages.length && !exemptHostNames.includes(window.location.href) ? (
```

Comment:
Will this catch if a user navigates to another page like the /History/ page? AFAIK the href includes the full URL, so that could result in a URL like `https://p5.sandbox.ai.huit.harvard.edu/dataset`. As an alternative, you could compare just the hostnames since that would cover everything on that domain (i.e. `p5.sandbox.ai.huit.harvard.edu` against `window.location.hostname`).

Reply:
This should only show up in the chat component since I think the `History` page is a different component.
Good point on the hostname vs the full URL!

Reply:
Good point -- if the banner is only rendered in the chat component then that won't matter.